src/pty.rs

```diff
@@ -0,0 +1,128 @@
+//! Create master and slave virtual pseudo-terminals (PTTYs)
+
+use std::ffi::CStr;
+use std::os::unix::io::RawFd;
+
+use libc;
+
+use {Error, fcntl, Result};
+
+/// Grant access to a slave pseudoterminal (see
+/// [grantpt(3)](http://man7.org/linux/man-pages/man3/grantpt.3.html))
+///
+/// `grantpt()` changezs the mode and owner of the slave pseudoterminal device corresponding to the
+/// master pseudoterminal referred to by `fd`. This is a necessary step towards opening the slave.
+#[inline]
+pub fn grantpt(fd: RawFd) -> Result<()> {
```

Comment:
But if you implement `Into` you also implement `From`, that's what my comment was about before. It's also bidirectional, so we're back to where we started. And you'd still need to use `.into()` everywhere since not all functions use `T: Into<RawFd>`. Since the topic of changing function signatures came up, there are 168 functions in `libc` that have fd's as inputs (as `c_int`, not `RawFd`) and 88 functions in `nix` that have `RawFd` inputs.